### PR TITLE
Moved countriesOfOrigin from Item to Content level in Deer model

### PR DESCRIPTION
**Pull request by dragospitica**

### CODE REVIEW

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/LegacyContentTransformer.java [hunk 1]
```diff
@@ -569,6 +568,8 @@ private BrandRef transformToBrandRef(org.atlasapi.media.entity.ParentRef ref,
         c.setYear(input.getYear());
         c.setLanguages(input.getLanguages());
 
+        c.setCountriesOfOrigin(input.getCountriesOfOrigin());
```
Comment (dragospitica) [inline atlas-legacy/src/main/java/org/atlasapi/system/legacy/LegacyContentTransformer.java]: This requires Owl model deployment first

File: atlas-core/src/main/java/org/atlasapi/content/Content.java [hunk 2]
```diff
@@ -56,6 +58,7 @@
     @Nullable private Set<Encoding> manifestedAs = Sets.newLinkedHashSet();
     private Boolean genericDescription = Boolean.FALSE;
     private ImmutableSet<EventRef> eventRefs = ImmutableSet.of();
+    private Set<Country> countriesOfOrigin = Sets.newHashSet();
```
Comment (tristanbeardmore-gray) [inline atlas-core/src/main/java/org/atlasapi/content/Content.java]: ImmutableSet here?

File: atlas-core/src/main/java/org/atlasapi/content/Content.java [hunk 3]
```diff
@@ -165,6 +168,19 @@ public void addEventRef(EventRef eventRef) {
         eventRefs = ImmutableSet.<EventRef>builder().add(eventRef).addAll(eventRefs).build();
     }
 
+    @FieldName("countries_of_origin")
+    public Set<Country> getCountriesOfOrigin() {
+        return countriesOfOrigin;
+    }
+
+    public void setCountriesOfOrigin(Iterable<Country> countries) {
+        this.countriesOfOrigin = Sets.newHashSet(countries);
```
Comment (tristanbeardmore-gray) [inline atlas-core/src/main/java/org/atlasapi/content/Content.java]: ImmutableSet.copyOf(countries)

File: atlas-core/src/main/java/org/atlasapi/content/Content.java [hunk 4]
```diff
@@ -183,6 +199,7 @@ public static Content copyTo(Content from, Content to) {
         to.manifestedAs = from.manifestedAs == null ? null : Sets.newLinkedHashSet(from.manifestedAs);
         to.genericDescription = from.genericDescription;
         to.eventRefs = from.eventRefs;
+        to.countriesOfOrigin = Sets.newHashSet(from.countriesOfOrigin);
```
Comment (tristanbeardmore-gray) [inline atlas-core/src/main/java/org/atlasapi/content/Content.java]: Probably also ImmutableSet.copyOf(from.countriesOfOrigin) here

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/LegacyContentTransformer.java [hunk 5]
```diff
@@ -578,22 +579,9 @@ private BrandRef transformToBrandRef(org.atlasapi.media.entity.ParentRef ref,
...
-                                input.country()
-                        );
-                    }
-
-                }
-        ));
-
-
+        c.setCertificates(input.getCertificates().stream()
+                .map(input1 -> new Certificate(input1.classification(), input1.country()))
```
Comment (jamiesprax) [inline atlas-legacy/src/main/java/org/atlasapi/system/legacy/LegacyContentTransformer.java]: name it gooderer

**Pull request merged**